scripts/lint-extras.sh

```diff
@@ -0,0 +1,93 @@
+#! /bin/bash
+# vim: set ts=4 sw=4 et :
+
+set -e
+
+# Run checks from root of the repo
+scriptdir="$(dirname "$(realpath "$0")")"
+cd "$scriptdir/.."
+
+# run_check <file_regex> <checker_exe> [optional args to checker...]
+# Pass empty regex when no regex is needed
+function run_check() {
+    regex="$1"
+    shift
+    exe="$1"
+    shift
+
+    if [ -x "$(command -v "$exe")" ]; then
+        if [ -z "$regex" ]; then
+            "$exe" "$@"
+        else
+            find . -path ./vendor -prune -o -regextype egrep -iregex "$regex" -print0 |
```

Comment:
maybe use `git ls-files --exclude='vendor/*' "${regex}"` instead of `find`

Reply:
this is not working, i will leave it as it is for now as this is copy-pasted from old script file